pyuploadcare/api_resources.py

```diff
@@ -231,24 +231,81 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
-        """Creates File copy
+    def copy(self, effects=None, target=None, make_public=None, store=None, pattern=None):
+        """Creates a File Copy on Uploadcare or Custom Storage. Will be deprecated in version 4.0.0"""
 
-        If ``target`` is ``None``, copy file to Uploadcare storage otherwise
-        copy to target associated with project.
-        Add ``effects`` to ``self.default_effects`` if any.
+        if target is not None:
```

Comment:
api_resources is full of inverted conditions. Maybe include in refactoring list to-do?